mozregression/main.py

```diff
@@ -240,6 +251,10 @@ def cli(argv=None):
         test_runner = CommandTestRunner(fetch_config, options.command,
                                         persist=options.persist)
 
+    prefs = mozprofile.prefs.Preferences()
+    if options.preferences:
+        prefs.add_file(options.preferences)
+
```

Comment:
Yep, that's right :) The idea is to be able to pass the arguments in the same way as in mozprofile, create a preferences object (and fill it accordingly to the passe arguments) then give that object to our launcher.